smartsim/_core/entrypoints/service.py

```diff
@@ -42,19 +42,21 @@ class Service(ABC):
     def __init__(
         self,
         as_service: bool = False,
-        cooldown: int = 0,
-        loop_delay: int = 0,
+        cooldown: float = 0,
+        loop_delay: float = 0,
         health_check_frequency: float = 0,
     ) -> None:
         """Initialize the ServiceHost
```

Comment:
Should `Initialize the ServiceHost` be `Initialize the Service`?

Reply:
yes. name change fail. fixed. found another instance of `ServiceHost` in `_on_start` and fixed there, too